apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/grpc/BridgeProtoTypeTranslator.java

```diff
@@ -75,6 +75,19 @@ public static String cqlTypeFromBridgeTypeSpec(
           frozen = udtType.getFrozen();
           break;
         }
+      case VECTOR:
```

Comment:
I guess I didn't realize how much of the type translation code is specific to the REST API, vs having something reusable in sgv2-quarkus-common. Is there a lot of duplicated code between REST API and JSON API?